### DISCUSSION TITLE
User expiry date variable

### DISCUSSION THREAD

**Discussion by hackmonker** [hrfee/jfa-go#81] [Ideas]
Can we get a user expiry variable too for welcome mails so that we can tell the user when their account will expire in case we use the expiry option

**Comment by hrfee** [comment]
Added the "yourAccountWillExpire" variable in 7a3e0d6.

**Reply by hackmonker** [reply]
awesome thanks!

**Reply by hackmonker** [reply]
> 
> 
> Added the "yourAccountWillExpire" variable in [7a3e0d6](https://github.com/hrfee/jfa-go/commit/7a3e0d60f95e5718109da36781de79128f04ab96).

How does jfa-go recognize the your invite will expire in line in the email?
I mean to say, If the invite does not have expiry set, then the line 

> your account will expire on

 should also be absent from the mail. is this possible? other wise the mail might not have the {userexpiry} variable but the line would be present.

**Reply by hrfee** [reply]
didn't even consider this, i'll try adding an if statement to the editor.

**Reply by hackmonker** [reply]
Thanks

**Reply by hrfee** [reply]
Added an if statement to the welcome email in 8fdab39, you can see it in the editor. Whatever you put inside 
`{if yourAccountWillExpire} your text here {endif}` will only be displayed if expiry is enabled.

**Reply by hackmonker** [reply]
Awesome. Thanks
